app-accessibility/sphinx2/sphinx2-0.6.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
 
 inherit autotools
```

Comment:
please drop USE=static-libs here (see https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0302), nothing depends on it for this pkg